python/lsst/afw/geom/skyWcs/skyWcsContinued.py

```diff
@@ -20,9 +20,80 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
-__all__ = []
+import numpy as np
 
+from lsst.utils import continueClass
 from ..python import reduceTransform
 from .skyWcs import SkyWcs
 
+__all__ = []
+
+
+@continueClass  # noqa: F811
+class SkyWcs:
+    def pixelToSkyArray(self, x, y, degrees=False):
+        """
+        Convert numpy array pixels (x/y) to numpy array sky (ra/dec)
+        positions.
+
+        Parameters
+        ----------
+        x : `np.ndarray`
+            Array of x points.
+        y : `np.ndarray`
+            Array of y points.
+        degrees : `bool`, optional
+            Return ra/dec arrays in degrees if True.
+
+        Returns
+        -------
+        ra : `np.ndarray`
+            Array of Right Ascension.  Units are radians unless
+            degrees=True.
+        dec : `np.ndarray`
+            Array of Declination.  Units are radians unless
+            degrees=True.
+        """
+        xy = np.vstack((x, y))
+        rd = self.getTransform().getMapping().applyForward(xy)
+        rd[0, :] = rd[0, :] % (2.*np.pi)
+
+        if degrees:
+            return np.rad2deg(rd[0, :]), np.rad2deg(rd[1, :])
```

Comment:
I'm not sure. but it may be more efficient to call `rad2deg` with `rd` first and then split them, and to me that seems a bit more elegant.

Reply:
Also, since `rd[0,:]` and `rd[1,:]` are used quite a few times, it may be better to name them `ra` and `dec`. Alternatively, you can use `np.vsplit` in the previous line and not define `rd` at all and ignore the comment above.